views/room/index.pug

```diff
@@ -4,16 +4,18 @@ include roomTemplate.pug
 
 block content
   h1(class='title') Tanulószobák
-  div(class='px-6 pt-3 md:mx-16 lg:mx-24 xl:mx-40')
-    if user && user.floor && user.floor >= 3 && user.floor <= 18
-      div(class='pt-3')
-        if busyRooms.some(it => it.room === user.floor)
-          - const group = busyRooms.find(it => it.room === user.floor)
-          +roomTemplate(user.floor, group, usageData.get(user.floor))
-        else
-          +roomTemplate(user.floor, false, usageData.get(user.floor))
-        div(class='h-px max-w-3xl mx-auto mt-8 mb-6 bg-gray-700 rounded')
-    div(class='py-2 space-y-8')
+  div(class='px-6 pt-3 xl:mx-52')
+    div(class='grid grid-cols-1 lg:grid-cols-2 gap-y-8 gap-x-8')
+      if user && user.floor && user.floor >= 3 && user.floor <= 18
+        div(class='pt-3')
+          if busyRooms.some(it => it.room === user.floor)
+            - const group = busyRooms.find(it => it.room === user.floor)
+            +roomTemplate(user.floor, group, usageData.get(user.floor))
+          else
+            +roomTemplate(user.floor, false, usageData.get(user.floor))
+    if user && user.floor
+      div(class='h-px max-w-3xl mx-auto mt-8 mb-6 bg-gray-700 rounded')
+    div(class='grid grid-cols-1 lg:grid-cols-2 gap-y-8 gap-x-8')
```

Comment:
I've been experimenting a little bit. Maybe we could use `md:grid-cols-2` in the first inner divs, they don't feel too bad on md breakpoint too.

Grid-cols-2 starting from lg (feels too spacious and wide):
![oie_kjT9tMXxlXhP](https://user-images.githubusercontent.com/60111772/120111943-1bb2c580-c174-11eb-87b2-c375952a85cc.png)

Grid-cols-2 starting from md:
![oie_Ik4ybDqP021g](https://user-images.githubusercontent.com/60111772/120111947-1e151f80-c174-11eb-8dff-3f2c84c8b4e3.png)

Sorry for bad cropping :(

Reply:
I like the more spacious version. Especially if any current / upcoming event is displayed.

Reply:
> I like the more spacious version. Especially if any current / upcoming event is displayed.

True, forgot about them. 👌